src/Collection.php

```diff
@@ -74,6 +74,15 @@ public function getData()
         return $this->_data;
     }
 
+    /**
+     * @param array $data
+     * @return Collection
+     */
+    public static function from(array $data = [])
```

Comment:
from can be confused with SQL `FROM` as this class may be used also in combination with AR. I suggest to rename it to `fromArray()`.

Reply:
Good point :smile: